automod_legacy/rules.go

```diff
@@ -184,6 +185,17 @@ func (i *InviteRule) Check(evt *discordgo.Message, cs *dstate.ChannelState) (del
 	return
 }
 
+type CachedInvite struct {
+	CreatedAt int64
```

Comment:
use time.Time as internally it uses a monotonic clock for clock for comparisons which is gonna be more reliable

Reply:
Alright! Will do